CodeModel/CaDETModel/CodeItems/CaDETClass.cs

```diff
@@ -42,34 +42,49 @@ public CaDETClass(string name, string fullName, string sourceCode): this(name)
 
         public List<CaDETClass> GetFieldLinkedTypes()
         {
-            return Fields.SelectMany(f => f.GetLinkedTypes()).ToList();
+            var instances = Fields.SelectMany(f => f.GetLinkedTypes()).ToList();
```

Comment:
"instances" is not a good word, as it is part of the DE domain and ubiquitous language, and not the code model.